Description: Prevent a failure of "make check" when stderr is not a tty.
 The original patch was partially applied for libcdio 0.82, but half the
 problem remains.
Author: Nicolas Boullis <nboullis@debian.org>
Last-Update: 2011-12-29
--- a/src/cd-paranoia/cd-paranoia.c
+++ b/src/cd-paranoia/cd-paranoia.c
@@ -352,7 +352,7 @@
     sector=inpos/CD_FRAMEWORDS;
     
     if(printit==-1){
-      if(isatty(STDERR_FILENO)){
+      if(isatty(STDERR_FILENO) || (logfile != NULL)){
 	printit=1;
       }else{
 	printit=0;
